test/index.test.js

```diff
@@ -1407,6 +1408,31 @@ const HasManyDissociator = require('../src/services/has-many-dissociator');
             .count();
           expect(count).toStrictEqual(4);
         });
+
+        it('should only return the ids of the other relationships when fields are specified', async () => {
+          expect.assertions(2);
+          const params = {
+            recordId: 100,
+            associationName: 'addresses',
+            fields: {
+              address: 'line,zipCode,city,country',
+              user: 'id',
+            },
+            page: { number: '1', size: '20' },
+            timezone: 'Europe/Paris',
+          };
+          const result = await new HasManyGetter(
+            models.user,
+            models.address,
+            sequelizeOptions,
+            params,
+          )
+            .perform();
+          expect(result[0]).not.toHaveLength(0);
```

Comment:
`toBeEmpty` ? Don't we have this library: https://github.com/jest-community/jest-extended ?

Reply:
Nope, it does not exist, and jest-extended is not part of dev dependencies